src/figuresClasses.ts

```diff
@@ -1,11 +1,86 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
create some type Color
```suggestion
  color: Color;
```